qaframework-bdd-tests/src/test/resources/features/refapp-2.x/05-location/locations.feature

```diff
@@ -25,8 +25,8 @@ Feature: Location Management
         And the user clicks on the delete forever icon
         Then the system deletes the location attribute type forever
 
-    @selenium
-    @location
+    #@selenium
+    #@location
```

Comment:
Any reason why you are commenting this feature file out.

Reply:
no worries i was just testing out one scenario but lemi do the needful